ui/DesignSystem/Component/EmptyState.svelte

```diff
@@ -46,17 +47,19 @@
   }
 </style>
 
-<div class="empty-state" data-cy="empty-state">
-  <Icon.EmptyState variant={icon} />
+<div class="empty-state" data-cy="empty-state" {style}>
+  <Icon.EmptyState variant={icon} style="height: 48px; width: 48px;" />
```

Comment:
This can be solved with `flex-shrink: 0;`. I wonder if we should just add that to all of the icons, so we don't bump into this issue anymore?

Reply:
Ok, I'll do that. FLEX MAGIC

Reply:
https://github.com/radicle-dev/radicle-upstream/pull/701/commits/517a433a57524ae0ad0ee9cfddb722ab458c6cb3